packages/clay-css/src/images/icons/spacer.svg

```diff
@@ -0,0 +1,7 @@
+<svg id="spacer" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
```

Comment:
hey @pat270 Spacer has some broken property... can you check it?

<img width="556" alt="screen shot 2019-02-21 at 13 58 16" src="https://user-images.githubusercontent.com/13750819/53187018-ef955b00-35e0-11e9-982b-947e17c969b6.png">

Reply:
I think we only have the class declaration twice in the `rect` tag.
```
class="lexicon-icon-outline spacer-indicator-start"
class="st0"
```